charts/gitops-server/templates/admin-user-roles.yaml

```diff
@@ -14,13 +14,13 @@ rules:
     verbs: [ "get", "list" ]
   - apiGroups: ["kustomize.toolkit.fluxcd.io"]
     resources: [ "kustomizations" ]
-    verbs: [ "get", "list", "patch" ]
+    verbs: [ "get", "list" ]
```

Comment:
Does sync or suspend use patch on these resources? At the moment, sync is available in the UI, while suspend is in the TODO list.

Reply:
Yes, it uses Patch.

https://github.com/weaveworks/weave-gitops/blob/134f2580fb73441705f9228b25b1cf1ff5f8dcf2/core/server/sync.go#L128

Reply:
Should that not use the users' permissions rather than the service account?

Reply:
It used the user's permissions because it is possible to have users with read access but can't resync (i don't know if it's handled on the UI or not).

Reply:
ignore me just spotted this is the admin user, not the service account